blog/2025/keycloak-devday-2026-announcement.adoc

```diff
@@ -0,0 +1,89 @@
+= Keycloak DevDay 2026 Announcement and Call-for-Papers
+:date: 2025-08-07
+:modified: 2025-08-07
+:publish: true
+:author: Sebastian Rose
+
+We (Niko and me (Sebastian)) are excited to announce the next edition of Keycloak DevDay!
+DevDay is expanding to a **2-day event** taking place again in Darmstadt, Germany on **March 5th and 6th, 2026**.
+
+The location is about 30 minutes away from Frankfurt/Main Airport by public transport, see https://www.keycloak-day.dev[] for details.
+
+== What's New for 2026?
+
+Keycloak DevDay - **the Keycloak community conference** - expands to exciting two days:
+
+**Day 1 (February 5th)** - Traditional Conference Format::
+Full day of talks, presentations, and panels featuring maintainers, industry-leading speakers, and community experts.
+Dive deep into Keycloak developments, best practices, real-world use cases, and the latest innovations in and around Keycloak.
+
+**Day 2 (February 6th)** - Unconference & Open Space::
+Interactive format shaped entirely by YOU, the participants! Join fellow Keycloakers in an Unconference/Open Space environment where you can:
+- Pitch your own topics and ideas
+- Lead discussions on challenges you're facing
+- Collaborate on solutions in small groups
+- Share knowledge through informal sessions
+- Network intensively with community members
+- Work on hands-on projects together
+
+This has been a successful pre-conf event in 2025.
+From the feedback of all DevDay participants we decided to make it an official part of the DevDay.
+
+== What to Expect
+
+DevDay 2026 continues our tradition of bringing together the vibrant Keycloak community with lots of opportunities for networking and exchange among like-minded people.
+As always, there will be plenty of drinks 🥤🍹 and food 🍔🌮🥗, as well as an exclusive surprise gift 🎁 for all participants.
+
+Meet and connect with:
+
+- Keycloak maintainers and core developers
+- Extension developers and contributors
+- Operators running Keycloak in production
+- Security architects and IAM specialists
+- Fellow community members from around the world
+
+== Call for Papers 📝
+
+The call for papers is now open: https://sessionize.com/keycloak-devday-2026! We're looking for talks covering:
```

Comment:
Is worth to highlight this line by making it bold (and italics)?

Reply:
Done